tests/run_standalone_tests.sh

```diff
@@ -69,7 +69,7 @@ for i in "${!parametrizations_arr[@]}"; do
   # execute the test in the background
   # redirect to a log file that buffers test output. since the tests will run in the background, we cannot let them
   # output to std{out,err} because the outputs would be garbled together
-  python ${defaults} "$parametrization" &>> standalone_test_output.txt &
```

Comment:
https://github.com/Lightning-AI/pytorch-lightning/pull/19141/files breaks TPU standalone tests. The TPU machines do not include the `python` to `python3` mapping

Reply:
if we do not keep breaking TPU tests we would see it comming